app/controllers/storage_items_controller.rb

```diff
@@ -0,0 +1,72 @@
+class StorageItemsController < ApplicationController
+  before_action :set_storage_item, only: %i[ show edit update destroy ]
+
+  # GET /storage_items or /storage_items.json
+  def index
+    @storage_items = StorageItem.all
+  end
+
+  # GET /storage_items/1 or /storage_items/1.json
+  def show
+    @storage_item = StorageItem.find(params[:id])
+    @place = @storage_item.place
+  end
+
+  # GET /storage_items/new
+  def new
+    @storage_item = StorageItem.new
+  end
+
+  # GET /storage_items/1/edit
+  def edit
+  end
+
+  # POST /storage_items or /storage_items.json
+  def create
+    @storage_item = StorageItem.new(storage_item_params)
+
+    respond_to do |format|
+      if @storage_item.save
+        format.html { redirect_to storage_item_url(@storage_item), notice: "Storage item was successfully created." }
+        format.json { render :show, status: :created, location: @storage_item }
+      else
```

Comment:
I don't think you need the `location` param to be set (you're already passing the @storage_item to the templates..)